release-notes/sc/_posts/2024-09-14-21.md

```diff
@@ -0,0 +1,51 @@
+#### Data statistics
+
+- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
+- This release contains **381** single cell RNA-Seq studies, consisting of **24,323,556** cells, of which **13,519,191** passed our QC from **21** different species.
+
+#### Software features
+- **Gut anatomogram** - a new human anatomogram for the large and small intestines has been added in this release and accompanies eligible experiments e.g. [E-GEOD-125970](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-125970/results/anatomogram)
+- **Inferred cell type marker gene heatmap** - for experiments where inferred cell type data is available you can now view marker genes based on the inferred cell type clusters e.g. [E-GEOD-227038](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-227038/results/marker-genes)
+
+#### New experiments
+Highlights
+- **Externally analysed data** - for the first time in this release we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon. 
```

Comment:
Adding punctuations to clarify sentence
```suggestion
- **Externally analysed data** - for the first time, in this release, we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon. 
```

Reply:
thank you, I re-phrased the sentence for greater clarity